index.html

```diff
@@ -555,54 +573,86 @@ <h2>Sample Policy Definitions</h2>
 &lt; NEL: {"max_age": 0}
         </pre>
 
-        <p>The above <a>NEL policy</a> provided in the server response contains <a>max_age</a> set to zero, which indicates that the user agent must delete the current registered <a>NEL policy</a> associated with the `example.com` <a>NEL origin</a> and all of its subdomains:</p>
-        <ul>
-          <li><a>include_subdomains</a> is implicit when <a>max_age</a> is zero</li>
-          <li><a>report_to</a> is optional when removing a previously registered policy</li>
-        </ul>
+        <p>
+        This <code>NEL</code> header instructs the user agent to remove any
+        existing <a>NEL policy</a> for <code>example.com</code>.
+        </p>
+
       </section>
       <section>
         <h2>Sample Network Error Reports</h2>
 
-        <p>This section contains an example network error report the user agent might queue when a network error is encountered for a <a>known NEL origin</a>.</p>
+        <p>
+        This section contains example <a>network error reports</a> the user
+        agent might queue when a network error is encountered for a <a>known NEL
+          origin</a>.  We show the full report payload that would be created by
+        the [[REPORTING]] API when uploading the report; the payload's

```

Comment:
The other instances of `[[REPORTING]]` in this document have a `!` in it: `[[!REPORTING]]`

Reply:
Done